components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -441,9 +442,10 @@ public class LegacyJobOrchestrator : IJobOrchestrator, IPublisherConfigServices,
         }
 
         /// <inheritdoc/>
-        public async Task<List<string>> PublishNodesAsync(PublishedNodesEntryModel request, CancellationToken ct = default) {
+        public async Task<PublishedNodesResponseApiModel> PublishNodesAsync(PublishedNodesEntryModel request, CancellationToken ct = default) {
```

Comment:
Let's keep the separation of abstractions here, so that the controller takes care of translation of response from here to `PublishedNodesResponseApiModel` and we work with `string` or `List<string>` here.

Reply:
Good point. Just changed.